crates/release_plz_core/src/release.rs

```diff
@@ -106,6 +108,9 @@ fn release_package(
     if input.dry_run {
         args.push("--dry-run");
     }
+    if input.gh_release {
+        args.push("--gh-release");
+    }
```

Comment:
These args are passed to `cargo`. `cargo` doesn't have the `--gh-release` flag. So this won't work I think. 🤔

Reply:
Why are you passing `-- --gh-release` to `cargo publish`?
This command invokes cargo to publish to crates.io, it doesn't invoke `release-plz`.
So you don't need to edit these args.